az-hadoop-jobtype-plugin/src/main/java/azkaban/jobtype/HadoopJavaJobRunnerMain.java

```diff
@@ -91,17 +91,23 @@ public void run() {
       }
     });
 
+    // Separate try catch for logger
     try {
-      _jobName = System.getenv(ProcessJob.JOB_NAME_ENV);
-      String jobPropsFile = System.getenv(ProcessJob.JOB_PROP_ENV);
-      String privatePropsFile = System.getenv(ProcessJob.JOBTYPE_PRIVATE_PROP_ENV);
-
       _logger = Logger.getRootLogger();
       _logger.removeAllAppenders();
       ConsoleAppender appender = new ConsoleAppender(DEFAULT_LAYOUT);
       appender.activateOptions();
       _logger.addAppender(appender);
       _logger.setLevel(Level.INFO); //Explicitly setting level to INFO
+    } catch (Exception e) {
+      throw e;
```

Comment:
Clarification: Do we wrap this logger init steps in exception to continue executing job even if the log set up fails? Should we not print in az log that we failed to init?

Reply:
Earlier everything was in one single try catch block. This separate out the logger in its own try catch. This way it can be used in catch block of the larger try catch.
We cant log anything anyway as we need a logger to begin with :)